eng/versioning/version_client.txt

```diff
@@ -297,7 +297,6 @@ com.azure.resourcemanager:azure-resourcemanager-videoanalyzer;1.0.0-beta.1;1.0.0
 # note: The unreleased dependencies will not be manipulated with the automatic PR creation code.
 # In the pom, the version update tag after the version should name the unreleased package and the dependency version:
 # <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
-unreleased_com.azure:azure-core-amqp;2.0.6
```

Comment:
Also, Since 2.0.6 is now released, update this line to 
 `com.azure:azure-core-amqp;2.0.5;2.0.6`  -> com.azure:azure-core-amqp;2.0.6;2.3.0-beta.3
After this change run the script to update the pom files

`python eng/versioning/update_versions.py --ut library --bt client --sr`

https://github.com/Azure/azure-sdk-for-java/blob/3c6b41432ef175ff82182a3f8c0cd011c5b2a20b/eng/versioning/version_client.txt#L61

Because We have already released 2.3.0-beta.2, next version will be 2.3.0-beta.3